homeassistant/components/switchbee/config_flow.py

```diff
@@ -0,0 +1,133 @@
+"""Config flow for SwitchBee Smart Home integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from switchbee.api import CentralUnitAPI, SwitchBeeError
+from switchbee.device import DeviceType
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.device_registry import format_mac
+
+from .const import CONF_DEFUALT_ALLOWED, CONF_DEVICES, CONF_SWITCHES_AS_LIGHTS, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Required(CONF_SWITCHES_AS_LIGHTS, default=False): cv.boolean,
```

Comment:
This seems to be weird? Home Assistant already provides this functionality out of the box? Why does this integration implement its own logic?

Reply:
CONF_SWITCHES_AS_LIGHTS will be removed